postcss.config.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   plugins: [
     require("stylelint")({
-      configFile: './.stylelintrc'
+      configFile: '../../.stylelintrc'
```

Comment:
Si, ma viene eseguito in ogni cartella dei pacchetti per come funziona lerna, quindi il comando parte da la!